boot-microservices/src/main/java/org/ow2/proactive/boot/microservices/IAMStarter.java

```diff
@@ -48,15 +48,15 @@
 
     private static final String SEPARATOR = File.separator;
 
-    private static Configuration config = new BaseConfiguration();
-
     private static Process process;
 
     private static List<String> command = new ArrayList<>();
 
     private static boolean started = false;
 
-    private static String iamURL;
+    public static Configuration config = new BaseConfiguration();
+
+    public static String iamURL;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this "public static iamURL" field final [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1444)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make iamURL a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)